lib/twilio-ruby/jwt/jwt.rb

```diff
@@ -0,0 +1,55 @@
+module Twilio
+  module JWT
+    class BaseJWT
+      def initialize(secret_key: nil, issuer: nil, subject: nil, algorithm: 'HS256', nbf: nil, ttl: 3600, valid_until: nil)
+        @secret_key = secret_key
+        @issuer = issuer
+        @subject = subject
+        @algorithm = algorithm
```

Comment:
I don't think it's a good idea for the algorithm to be configurable, anywhere. This is the root of pretty much every JWT vulnerability. There's also no reason to use any algorithm but HS256.